stable/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 1.4.1
+version: 1.5.0
```

Comment:
This should be a major bump due to all the label changes.

Reply:
raised to 2.0.0